### ISSUE TITLE
Add busy channel test data

### ISSUE THREAD

**Issue by apcarp** [JeffersonLab/jmyapi#20]
We need a channel that is really busy to conduct effective mySamplerStream tests.  The container is already a little slow to start, and I suspect this will also slow it down more.

**Comment by apcarp** [comment]
I tested a data generation script.  It was very slow.  I limited the mysql command line length to avoid issues with exceeding the maximum shell line length, and I this resulted in needing to make hundreds of mysql connections to generate the data.  The result was that it took several minutes to generate and insert a few hours of busy data.

I also tested a using a compressed SQL file for importing the data.  This seems to be faster than using a data generating script as ~1.25 days worth of data was inserted in tens of seconds.  The compressed SQL file is only several MB, which is larger than standard source files, but I don't think prohibitively large.  I'm will go this route in my next PR as it is needed for testing MySamplerStream improvements.
